tools/build_variables.bzl

```diff
@@ -360,6 +360,7 @@ lazy_tensor_core_sources = [
     "torch/csrc/lazy/core/ir.cpp",
     "torch/csrc/lazy/core/ir_metadata.cpp",
     "torch/csrc/lazy/core/ir_util.cpp",
+    "torch/csrc/lazy/tensor/permutation_util.cpp",
```

Comment:
@wconstab I put the new file in a new subdirectory. Want to hear what's your plan in organizing files here. Do you prefer putting everything under `core`? Or is `core` just for IR nodes?

Reply:
I don't have a grand plan;  I think originally I did a little planning and came up with /core and /backend as two folders.  Currently I'm not sure I like it anymore; In general I think I'd lean towards putting everything into one folder first, then reorganizing it once we have enough files to really justify a separate folder.  

What do you think about deleting the 'core' folder entirely and just putting everything under torch/lazy for now?  Except for a separate torch/lazy/ts folder for the backend impl?
torch/lazy (most stuff here)
torch/lazy/ts_backend/ (backend impl)

Or, do you like core and maybe
torch/lazy/core/ (everything else)
torch/lazy/backend/ (backend interface)
torch/lazy/ts_backend (backend implementation)

Reply:
I like keeping torch/lazy/core/ as a subdirectory. Let me move the new files into the same directory.